File: archive/logic/TuringMachineSimulationNew/Tapes/TapesUtilities/TapeSymbols.cs
namespace TuringMachineSimulation.Logic.Tapes.TapesUtilities
{
    public static class TapeSymbols
    {
        /// <summary>
        /// Empty content, the default value of the tape <br/> Notes: the tape doesn't actually store it
        /// </summary>
        public const char Blank = '_';
        
        /// <summary>
        /// Start symbol, for left limited tapes
        /// </summary>
        public const char Start = '>';
        
        /// <summary>
        /// End symbol, for right limited tapes
        /// </summary>
        public const char End = '<';
        
        /// <summary>
        /// None symbol, for read/write only heads
        /// </summary>
        public const char None = '/';
    }
}

File: frontend/turing-machine-web/src/logic/Tapes/TapesUtilities/TapeSymbols.ts
export class TapeSymbols {
    /**
     * Empty content, the default value of the tape. Notes: the tape doesn't actually store it
     */
    static readonly Blank: string = '_';


    /**
     * Start symbol, for left limited tapes
     */
    static readonly Start: string = '>';


    /**
     * End symbol, for right limited tapes
     */
    static readonly End: string = '<';

    
    /**
     * None symbol, for read/write only heads
     * Note: this symbol might be dangerous, but the system won't actually write this symbol
     */
    static readonly None: string = '/';


    /**
     * Running symbol, the machine that reads this resumes from pause
     * Note: Use symbol that won't appear in normal content
     */
    static readonly Running: string = '\u0001';


    /**
     * Running symbol, the machine that reads this resumes from pause
     * Note: Use symbol that won't appear in normal content
     */
    static readonly Pause: string = '\u0002';
}